### ISSUE TITLE
Range matching patterns?

### ISSUE THREAD

**Issue by gvanrossum** [gvanrossum/patma#12]
Some languages have range cases, so you could write e.g. `case 1...6: print("Dice")`. This seems tricky, and quite specialized. Let's not do this?

**Comment by thautwarm** [comment]
Yes, implementing range patterns with dedicated syntax is a waste. We can implement this with current protocol.

**Comment by gvanrossum** [comment]
But we should not. I have seen your hack and think it is too obfuscated.

**Comment by thautwarm** [comment]
If you can support parameterised patterns in some way, it won't be obfuscated at all.
